src/main/java/com/squareup/kotlinpoet/CodeBlock.kt

```diff
@@ -374,6 +397,7 @@ class CodeBlock private constructor(
     @JvmField internal val LOWERCASE = Regex("[a-z]+[\\w_]*")
     internal const val ARG_NAME = 1
     internal const val TYPE_NAME = 2
+    internal val NO_ARG_PLACEHOLDERS = setOf("%W", "%<", "%>", "%[", "%]")
```

Comment:
nit: reorder `%<` and `%>` ? Typically we indent with `>` then unindent with `<` not the opposite!